src/components/popup.tsx

```diff
@@ -28,46 +30,38 @@ const Popup = () => {
       }
 
       setActiveColor(result);
-      setShowCustomInput(!colors.includes(result));
     }
 
     getActiveColorEffect();
   }, []);
 
   return (
     <div className="colorLinks">
-      <span
+      <h1
```

Comment:
Preserve the nice visual indicator for the current color, but also let's remind users what extension they're using.